src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
Actually, I am having problems with `npm run build:aot` but I am having them also in the `master` branch... 

It looks something like
```
Angular compilation done, starting webpack bundling.
Error: undefined, resolving symbol OpaqueToken in /*****/ng2-smart-table/node_modules/@angular/core/src/di/opaque_token.d.ts
...
```
Do you have any hint?